### ISSUE TITLE
Is there a new docs site?

### ISSUE THREAD

**Issue by iennae** [gomods/athens#1389]
**Describe the bug**
Going through the repository seeing links to the online docs site, I get server IP can not be found errors. It looks like the whois information was recently updated "Updated Date: 2019-09-17T21:20:29Z". 

**Error Message**
ERR_NAME_NOT_RESOLVED

**To Reproduce**
Browse to website: https://docs.gomods.io/contributing/community/philosophy/ 

**Expected behavior**
I expect to find a website :)

**Comment by awkj** [comment]
After I spent thirty minutes to fuck China GFW , I found that the website was hanging, very funny.

**Comment by iwittkau** [comment]
Looks like a DNS problem?

```
$ nslookup gomods.io 1.1.1.1
Server:		1.1.1.1
Address:	1.1.1.1#53

** server can't find gomods.io: SERVFAIL

$ nslookup gomods.io 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

** server can't find gomods.io: SERVFAIL
```

**Comment by iennae** [comment]
@iwittkau yeah that's where my ERR_NAME_NOT_RESOLVED comment in the error message was from :)

**Comment by iennae** [comment]
Looks like this has been fixed! Thanks.

**Comment by arschles** [comment]
Sorry about this, folks. Here's what happened: 

We use Netlify to host our docs site, which requires that we set nameservers to them. When namecheap took over the records, they also overrwrote the nameservers, so no more Netlify.

The fix: simply set nameservers to Netlify and wait for TTL expiry around the world.

Why I did this: I did this change so that the core maintainers can share admin rights to the gomods.io domain
